internal/verifier/change_stream.go

```diff
@@ -52,8 +53,79 @@ func (uee UnknownEventError) Error() string {
 	return fmt.Sprintf("received event with unknown optype: %+v", uee.Event)
 }
 
+type ChangeStreamReader struct {
+	readerType whichCluster
+
+	lastChangeEventTime *primitive.Timestamp
+	logger              *logger.Logger
+	namespaces          []string
+
+	metaDB        *mongo.Database
+	watcherClient *mongo.Client
+	clusterInfo   util.ClusterInfo
+
+	changeStreamRunning  bool
+	ChangeEventBatchChan chan []ParsedEvent
+	WritesOffTsChan      chan primitive.Timestamp
+	ErrChan              chan error
+	DoneChan             chan struct{}
```

Comment:
nit: Do these need to be exported?

Reply:
Technically none of them need to be exported since they're all in the `verifier` package. Changed to private.